src/tools/MacOS Installation.md

```diff
@@ -0,0 +1,47 @@
+# MacOS Installation
+
+### 1. Download
+
+Download SlimeVR for Mac (the .dmg file from (https://github.com/SlimeVR/SlimeVR-Server/releases/latest/download/SlimeVR-mac.dmg)
+
+### 2. Install
+
+Open the .dmg and drag SlimeVR app into Applications.
+
+### 3. Open
+
+Open SlimeVR from Applications. (You will get this error shown below)
+
+
+<img width="268" alt="warning" src="https://github.com/user-attachments/assets/018db046-56bc-4b74-9104-5a92e8d77408" />
+
+### 4. Security
+
+Go to System Settings - > Privacy and Security -> You should see SlimeVR was blocked as shown below. Click Open Anyway. 
+
+
+<img width="268" alt="warning" src="https://github.com/user-attachments/assets/70e4d0b8-66b8-4e43-9358-b1bb6d8a2701" />
+
+
+# Installing Java
```

Comment:
There should only be one title header (`#`) in the page, for any sub-headers, start with a secondary header (`##`).

Reply:
I have fixed these for now